docs/contributing.md

```diff
@@ -74,22 +74,21 @@ $ hatch run  pre-commit install
 To run the tests, use:
 
 ```shell
-$ hatch run test:test
+$ hatch test
 ```
 
-Because Edgy uses pytest, any additional arguments will be passed. More info within the
-[pytest documentation](https://docs.pytest.org/en/latest/how-to/usage.html)
-
-For example, to run a single test_script:
+To run a single test_script:
 
 ```shell
-$ hatch run test:test tests/test_apiviews.py
+$ hatch test tests/test_apiviews.py
```

Comment:
Not apiviews. Let's put a real existing file. This was based on Esmerald by that time.